### PR TITLE
chore(i18n,docs): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

File: docs/i18n/portuguese/how-to-work-on-coding-challenges.md [hunk 1]
```diff
@@ -432,8 +435,8 @@ function myFunc() {
 
 #### Code Explanation
 
-- Code explanation goes here
-- Code explanation goes here
+- A explicação do código fica aqui
```
Comment (texas2010) [inline docs/i18n/portuguese/how-to-work-on-coding-challenges.md:438]: I am not sure why this double line has the same message.

File: docs/i18n/portuguese/how-to-work-on-coding-challenges.md [hunk 2]
```diff
@@ -349,80 +352,80 @@ class MyComponent extends React.Component {
...
 
-This summarizes what needs to be done without just restating the challenge description and/or instructions. This is an optional section
+Resume o que precisa ser feito sem copiar a descrição do desafio e/ou instruções. Essa é uma seção opcional
 
-#### Relevant Links
+#### Links relevantes
 
-- [Link Text](link_url_goes_here)
-- [Link Text](link_url_goes_here)
+- [Texto do link](url_do_link_fica_aqui)
```
Comment (texas2010) [inline docs/i18n/portuguese/how-to-work-on-coding-challenges.md:409]: I am not sure why this double line has the same message.

File: docs/i18n/portuguese/how-to-work-on-coding-challenges.md (hunk 1 above)
Comment (nhcarrigan) [reply docs/i18n/portuguese/how-to-work-on-coding-challenges.md:438]: This one could probably be fixed, yeah (in the English file via PR)

File: docs/i18n/portuguese/how-to-work-on-coding-challenges.md (hunk 2 above)
Comment (nhcarrigan) [reply docs/i18n/portuguese/how-to-work-on-coding-challenges.md:409]: This is correct, though - the template is for adding new Guide posts, and is showing how to structure the list of relevant links.

**Review by naomi-lgbt — APPROVED** [review]

File: docs/i18n/portuguese/devops.md [hunk 3]
```diff
@@ -106,27 +106,27 @@ Currently, only members on the developer team can push to the production branche
...
+O processo é quase o mesmo das plataformas de preparo, com algumas verificações extras. This is just to make sure, we do not break anything on freeCodeCamp.org which can see hundreds of users using it at any moment.
 
-| Do NOT execute these commands unless you have verified that everything is working on the staging platform. You should not bypass or skip any testing on staging before proceeding further. |
-|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-|                                                                                                                                                                                            |
+| Do NOT execute these commands unless you have verified that everything is working on the staging platform. Você não deve ignorar qualquer teste na fase de preparo antes de prosseguir. |
```
Comment (texas2010) [inline docs/i18n/portuguese/devops.md:117]: I am not sure about this line. Does this message need to be translated?

File: docs/i18n/portuguese/how-to-work-on-coding-challenges.md (hunk 2 above)
Comment (texas2010) [reply docs/i18n/portuguese/how-to-work-on-coding-challenges.md:409]: @nhcarrigan, oh ok, I just saw two same example markdown links if you are fine with that.

File: docs/i18n/portuguese/devops.md (hunk 3 above)
Comment (nhcarrigan) [reply docs/i18n/portuguese/devops.md:117]: Unlike the curriculum, our documentation is set to allow partially translated files.

**Review by texas2010 — APPROVED** [review]
Look good to me. just some lines need to fix in the English file. 👍🏻

**Pull request merged**